src/google-classroom-extractor/google_classroom_extractor/main.py

```diff
@@ -35,36 +42,51 @@
     USER_ACTIVITY_ROOT_DIRECTORY,
 )
 
+logger: logging.Logger
 
-def request() -> Result:
-    load_dotenv()
+
+def configure_logging():
+    global logger
+
+    logger = logging.getLogger(__name__)
+
+    level = os.environ.get("LOGLEVEL", "INFO")
```

Comment:
I added a little more organization in this file. Schoology needs it too, but it would be more invasive than the other little refactors I've done, so I don't want to tackle that in this ticket.